worlds/oot_soh/Enums.py

```diff
@@ -3548,7 +3548,7 @@ class Events(StrEnum):
     SHADOW_TEMPLE_COMPLETED = "Shadow Temple Completed"
     TIME_TRAVEL = "Time Travel"
     FREED_EPONA = "Freed Epona"
-    RESCUED_ALL_CARPENTERS = "Rescued All Carpenters"
+    RESCUED_ALL_CARPENTERS = "GF Freed All Carpenters"
```

Comment:
This one isn't right. This is an event, not a location. You've already renamed the actual Location to `GF_GERUDO_MEMBERSHIP_CARD = "GF Freed All Carpenters"` and inside `carpenters_location_table`